packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -352,13 +356,24 @@ class ChromiumLauncher {
     try {
       if (sourceChromeDefault.existsSync()) {
         targetChromeDefault.createSync(recursive: true);
-        copyDirectory(sourceChromeDefault, targetChromeDefault);
+        copyDirectory(
+          sourceChromeDefault,
+          targetChromeDefault,
+          shouldCopyDirectory: _isNotCacheDirectory,
+        );
       }
     } on FileSystemException catch (err) {
       _logger.printError('Failed to restore Chrome preferences: $err');
     }
   }
 
+  // Cache, Code Cache, and GPUCache are nearly 1GB of data
+  bool _isNotCacheDirectory(Directory directory) {
```

Comment:
Can this be an allowlist instead of denylist? Are there any docs from Chrome that this could link to to monitor for removals/additions to this list?

Reply:
I agree that allowlist would be a better solution, but significantly more time consuming to maintain. Unfortunately I wasn't able to find much in terms of docs here (CodeCache is CodeCacheHost, which seems to be storing cached compiled metadata. Gpu cache is something similar.

I don't have a great idea of what is in Cache itself, but imo it doesn't _really_ matter. The original intention of this copying cache behavior was to preserve things like browser window size and local storage - but I don't think we can reasonably spend ~15 seconds blocking at startup/teardown to support this. Either the current profile information we copy is sufficient, or we should direct users to use a different dev workflow, instead of slowing everyone down.

We could of course remove the caching behavior altogether, though I think that might be more noticeable

Reply:
Is there a good place to put this information in a comment or comments in the code?

Reply:
I added some more details above on _cacheUserSessionInformation